tests/publiccloud/xfsprepare.pm

```diff
@@ -91,11 +128,17 @@ sub run {
     $self->select_serial_terminal;
 
     record_info("lsblk", script_output("lsblk"));    # debug output
+
+    # Special 'nvme' device name means get the first not-used nvme disk.
+    # This is necessary on EC2-ARM instances, where the secondary nvme system disk is not predictable (can be nvme0n1 or nvme0n1)
```

Comment:
"nvme0n1 or nvme0n1" please change one of them

Reply:
Good catch!

Reply:
done